Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.



Generated by build [OMERO-DEV-latest-docs-autogen#1078](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1078/).

----
--no-rebase

hflynn commented:
Looks good but I'll re-run the merge build once it's green

hflynn commented:
https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/871/ green with this included and History renders as expected on built pages.

Provided @pwalczysko is happy that the OS walkthrough changes are expected, this should be good to merge.

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#1079 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1079/)

jburel commented:
The mac OS walkthrough is no longer used i.e. we do not use any "command snippets" in the doc
As mentioned previously we have not updated the files in omero-install to reflect the changes in the doc.
Conclusion: the changes will have no effect and the PR can be merged
cc @sbesson
